packages/e2e/e2e_macos/macos/Classes/E2EPlugin.m

```diff
@@ -0,0 +1,20 @@
+#import "E2EPlugin.h"
```

Comment:
Optional nit: Swift is the preferred language for macOS plugins, and this plugin is trivial (you should be able to copy the example plugin class and just change the class name, the channel string, and add the allTestsFinished handler), so it would be nice to make this Swift rather than ObjC.

Reply:
Done